packages/protocol/test/stability/sortedoracles.ts

```diff
@@ -253,58 +256,79 @@ contract('SortedOracles', (accounts: string[]) => {
   })
 
   describe('#report', () => {
-    function expectedNumeratorFromGiven(
+    function expectedMedianRateFromGiven(
       givenNumerator: number | BigNumber,
       givenDenominator: number | BigNumber
     ): BigNumber {
-      return expectedDenominator.times(givenNumerator).div(givenDenominator)
+      return toFixed(new BigNumber(givenNumerator).dividedBy(new BigNumber(givenDenominator)))
     }
 
-    const numerator = 10
+    const numerator = 5
```

Comment:
No need to define these as numerator and denominator now, you can just use a single "rate"